support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx

```diff
@@ -20,46 +26,106 @@ const threeTierTerms = css`
 	}
 `;
 
+export const productNameUSTier3 = 'Guardian Weekly and All-Access Digital';
```

Comment:
Could / should we use `Three` over `3`. By following convention we'll make find/replace refactoring easier for future folks.

Reply:
Maybe we could drop this value into the `productCatalog` file?

Reply:
moved this across to `productCatalog` file